demo/js/components/victory-candlestick-demo.js

```diff
@@ -78,56 +78,25 @@ export default class App extends React.Component {
 
     return (
       <div className="demo" style={containerStyle}>
-        <svg height={500} width={500}>
+        <VictoryChart
+          scale={{ x: "time" }}
+          style={style}
+          domainPadding={{ x: [20, 50] }}
+        >
           <VictoryCandlestick
-            style={{ data: { width: 10 }, parent: style.parent }}
-            labels={({ datum }) => `x: ${datum.x.getDate()}`}
-            labelOrientation={{ low: "bottom", high: "top" }}
-            openLabels={({ datum }) => datum.open}
+            aria-label="Victory Candlestick with Victory Chart wrapper"
+            candleColors={{ positive: "#8BC34A", negative: "#C62828" }}
             closeLabels={({ datum }) => datum.close}
-            lowLabels={({ datum }) => datum.low}
-            highLabels={({ datum }) => datum.high}
+            data-test-variable="TESTING 123"
             data={data}
+            highLabels={({ datum }) => datum.high}
+            labelOrientation={{ open: "top", high: "top" }}
+            lowLabels={({ datum }) => datum.low}
+            openLabels={({ datum }) => datum.open}
             size={8}
-            standalone={false}
-            events={[
```

Comment:
Another option (probably in addition to the tests if we need it) would be to add an additional accessibility demo to each component, rather than updating existing demo components. That would be useful for accessibility audits, etc.